src/components/field-label/field-label.js

```diff
@@ -33,13 +33,20 @@ export const FieldLabel = props => (
       </Spacings.Inline>
 
       {props.hint && (
-        <Spacings.Inline alignItems="center" scale="xs">
-          {props.hintIcon &&
-            // FIXME: add proper tone when tones are refactored
-            React.cloneElement(props.hintIcon, {
-              size: 'medium',
-              theme: props.hintIcon.props.theme || 'orange',
-            })}
+        <Spacings.Inline alignItems="center" scale="s">
+          {props.hintIcon && (
+            <div
+              css={css`
+                display: flex;
+              `}
+            >
+              {React.cloneElement(props.hintIcon, {
+                // FIXME: add proper tone when tones are refactored
+                size: props.hintIcon.props.size || 'medium',
```

Comment:
I was supposed to talk to you first about this...
@filippobocik decided against allowing consumers to allow this.